requirements.txt

```diff
@@ -8,7 +8,7 @@ pyaml==19.4.1
 -e git+https://github.com/flupke/pypotrace.git@76c76be2458eb2b56fcbd3bec79b1b4077e35d9e#egg=pypotrace
 pyyaml==5.3.1
 requests==2.24.0
--e git+https://github.com/bsdz/rmapy.git@d53983e68021402f6b1dc8d736633a9c53f903b2#egg=rmapy
+-e git+https://github.com/j6k4m8/rmapy.git@d53983e68021402f6b1dc8d736633a9c53f903b2#egg=rmapy
```

Comment:
Why switch from my fork? My fork has been tested and works at least for myself and a few others.

Reply:
Right, this isn't necessary - I thought it was a problem with the fork when I kept getting stuck at Not Authenticated Exceptions and happened to fix it when I did renew_token(). Just tested with your fork and it works fine, sorry about that.